app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/popup/PopupView.kt

```diff
@@ -154,6 +162,10 @@ class PopupView : FrameLayout {
             binding.root.popupFieldEditValue
         }
 
+        val popupFieldCvdValues: Spinner by lazy {
```

Comment:
naming suggestion: `codedValueDomainSpinner`:
- remove redundant term `popupField`
- spell out codedValueDomain instead of abbreviation
- add the UI type as a suffix ("Spinner"), 

I would apply the same to the other properties, e.g. `popupFieldLabel` -> `labelTextView` etc...

I think it would make the code more readable in places where these properties are referenced.

Reply:
Updated the names